doc/developer/building-libyang.rst

```diff
@@ -3,6 +3,8 @@ support. Unfortunately, most distributions do not yet offer a ``libyang``
 package from their repositories. Therefore we offer two options to install this
 library.
 
+On Fedora 33+ it can be installed with `sudo dnf install libyang libyang-devel`.
```

Comment:
Why not include this fedora-specific info on the fedora doc page?

Reply:
It is on F24+ while this only works with F33+ and the libyang topic is introduced after the basic dependency section.

Reply:
so that's sort of what I'm suggesting: we have a page for fedora. why not just add this line before the common libyang section, noting that for fedora33+ there's a dnf version?

Reply:
Ok.

Another question would be whether F24 is still important. We could raise it to F33 and just add the dependency to the list. F33 is current stable and it shouldn't be far from F34 release.

Reply:
seems like a reasonable thing to do.

Reply:
well, we support (or try to support) a wide-ish range of distributions and versions - so if the basic fedora instructions don't need to change from release to release, or unless someone wants to make a proposal to fix/correct/or remove something, the dev doc sort of stays stable. I don't have a strong feeling either way - I just was suggesting that fedora-specific stuff should be in the fedora doc.

Reply:
I've pushed a proposal.

Reply:
fedora themselves only support last 2 released versions so it makes sense we do the same I think.